diff --git a/recipes-core/images/upboard-image-base.inc b/recipes-core/images/upboard-image-base.inc
index 8343080..915256d 100644
--- a/recipes-core/images/upboard-image-base.inc
+++ b/recipes-core/images/upboard-image-base.inc
@@ -2,7 +2,7 @@ IMAGE_FEATURES += "package-management"
 
 IMAGE_INSTALL += "kernel-modules"
 IMAGE_INSTALL += "dmidecode"
-IMAGE_INSTALL += "python"
+#IMAGE_INSTALL += "python"
 IMAGE_INSTALL += "python3"
 IMAGE_INSTALL += "python3-pip"
 IMAGE_INSTALL += "alsa-lib alsa-utils alsa-tools"
@@ -14,5 +14,6 @@ IMAGE_INSTALL += "picocom"
 IMAGE_INSTALL += "connman connman-client"
 
 DISTRO_FEATURES_append = " bluez5 bluetooth wifi"
-IMAGE_INSTALL_append += " bluez5 systemd"
+#IMAGE_INSTALL_append += " bluez5 systemd"
+IMAGE_INSTALL_append += " bluez5"
 IMAGE_INSTALL_append += " hostapd dhcp-client iw wpa-supplicant"
